app/src/androidTest/java/org/simple/clinic/overdue/AppointmentRepositoryAndroidTest.kt

```diff
@@ -2824,7 +2904,7 @@ class AppointmentRepositoryAndroidTest {
             dateOfBirth = patientProfile.patient.dateOfBirth,
             age = patientProfile.patient.age,
             appointment = appointment,
-            phoneNumber = patientProfile.phoneNumbers.first(),
+            phoneNumber = if (patientProfile.phoneNumbers.isEmpty()) null else patientProfile.phoneNumbers.first(),
```

Comment:
Kotlin stdlib has a extension we can use called `firstOrNull` that does the same thing. Let's use that.